jenkins-pr-collector.go

```
@@ -352,63 +415,132 @@ func fetchJenkinsPluginInfo(updateCenterURL string) (map[string]PluginInfo, erro
 	return pluginRepos, nil
 }
 
-// ExecuteGraphQL executes a GraphQL query against the GitHub API
-func (c *GraphQLClient) ExecuteGraphQL(ctx context.Context, query string, variables map[string]interface{}, result interface{}) error {
-	// Create request
-	reqBody, err := json.Marshal(GraphQLRequest{
-		Query:     query,
-		Variables: variables,
-	})
+// Add isRetryableError function to check if an error should be retried
+func isRetryableError(err error) bool {
```

golangci-lint (1.62.2)

func `isRetryableError` is unused

(unused)